src/en/space-station-14/departments/medical/proposals/Virology Reworked.md

```diff
@@ -0,0 +1,72 @@
+# Virology Reworked
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| AdmiralObvious | :x: | TBD |
+
+## Overview 
+Virology existed in the past and for the most part was very static and boring, as well as generally unpleasant to deal with. This proposal is hopefully going to make Virology as a job more interesting, as well as have more impact on a round. Viruses would be procedural instead of one and done.
+
+## Viruses
+At the start of a round, a large percentage of the crew should start with some sort of benign effect virus (will explain more later about benign versus active viruses). Each crewmember would start with between 0 and 3 viruses in their system.
+These viruses initially won't have any effects to the crew carrying said viruses, but later as the round progresses can mutate based on various factors.
+Viruses can be identified and tracked using a similar system to the detective DNA sequencing and collected by crew using sterile swabs on individuals, or by taking blood samples from crew. Method of sample collection will determine which, if any viruses are collected. These samples can then be inserted into the medical diagnoser (or something else but we have the machine for it) to determine the 6 core characteristics of the virus.
+
+## Viral Characteristics 
+Viruses will have 4 of 6 characteristics that the virologist can detect using the diagnoser. Symptoms, which can have many potential symptoms the virologist can see to assess who has what disease potentially.
+Volatility, which would operate on a sliding scale of how likely the virus is to mutate, and how severe the potential mutations can be using a random weighted point buy, similar to botany chem mutations.
+Estimated lifespan of the virus before it eventually "dies", by 20 minutes, none of the initial viruses should exist.
```

Comment:
What about late joins, do they also spawn with fresh benign viruses? I think that would fit the flavor of the mechanic.

Reply:
That's a good idea, I'll look into adding that on my next major pass.